src/components/App/index.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
```

Comment:
Does App as a new component not get tests?

Reply:
It's not new so much as moved. I can add tests, but in the interests of not ballooning how much time I spent on this, I did not.

Reply:
https://openedx.atlassian.net/browse/ENT-4046